doc/rtd/topics/tutorials/qemu-debugging.rst

```diff
@@ -0,0 +1,46 @@
+.. _debug information:
+
+Qemu Tutorial Debugging
+***********************
+
+You may wish to test out the commands in this tutorial as a `script`_
+to check for copy-paste mistakes.
+
+If you successfully launched the virtual machine, but couldn't log in,
+there are a few places to check to debug your setup.
+
+To debug, answer the following questions:
+
+Did cloud-init discover the imds webserver?
+===========================================
+
+The webserver should print a message in the terminal for each request it
+receives.  If it didn't print out any messages when the virtual machine booted,
+then cloud-init was unable to obtain the config. Make sure that the webserver
+can be locally accessed using ``curl`` or ``wget``.
+
+.. code-block:: sh
+
+   $ curl 0.0.0.0:8000/user-data
+   $ curl 0.0.0.0:8000/meta-data
+   $ curl 0.0.0.0:8000/vendor-data
+
+
+Did the imds webserver serve the files it was expected to serve?
+================================================================
+
+When launching Qemu, if the webserver prints out 404 errors, then try to figure
+out why those files can't be served.
+
+Did you forget to start the server in the temp directory?
+
+
+Were the configurations inside of the file correct?
```

Comment:
```suggestion
Were the configurations inside the file correct?
```

Reply:
Also, just want to double check what file "the file" refers to here?

Reply:
Good point. This refers to the user-data configuration file. I'll add that.